core/http/src/main/java/org/trellisldp/http/impl/BaseLdpHandler.java

```diff
@@ -66,6 +58,7 @@ protected BaseLdpHandler(final TrellisRequest request, final ServiceBundler serv
         this.requestBaseUrl = getRequestBaseUrl(request, baseUrl);
         this.request = request;
         this.services = services;
+        services.getConstraintServices().forEach(constraintServices::add);
```

Comment:
Can we not just take a reference to the constraint services and use them from that reference?

Reply:
Definitely. I just updated the PR to do just that.